### DISCUSSION TITLE
The finish isn't so wonderful

### DISCUSSION THREAD

**Discussion by SteveBz** [1coderookie/KobraGoNeoInsights#3] [General]
I'd say it's great that it prints straight out of the box.  Fantastic.

But the finishes aren't alway so wonderful.  The bottom side especially is very rough and I'm not sure what to do about that,  It seems wasteful to put tape on the magnetic PEI board, but I could do.  Also maybe my z-offset needs adjusting.

The sides and the top aren't great either.  My modified old Startt/Tronxy XY100 had a better finish.

What do you think?

Kind regards

Steve.

**Comment by 1coderookie** [comment]
Do you mean this kind of 'rippled' surface the PEI plate causes at the bottom of the first layer? Or what exactly do you mean with the 'finish'? I also don't know what you mean with sides and top - maybe you wanted to put a picture in here as well to show me something? Or what are you actually referring to..?

**Comment by 1coderookie** [comment]
About the bottom of the 1st layer: so if your z-offset is right and there are'nt any gaps between the lines, then it should like 'one surface' over all. And then you'll see the texture of the PEI plate, that's correct. It doesn't actually bother me personally tho. If you absolutely don't like that, you plrobably want to use a glass bed instead of the PEI plate. I personally wouldn't suppose to use tape on the plate anyway..

**Comment by 1coderookie** [comment]
Sides and top finish/quality also depends on your nozzle size, layer hight, a flawless motion of the axes and the setting for the top layer in the slicer btw.

**Comment by SteveBz** [comment]
By finish I mean the exposed surfaces, the speckly bottom, the sides and the top.

Bottom:
![WhatsApp Image 2023-03-05 at 14 28 14](https://user-images.githubusercontent.com/1646078/222966709-0e69beb3-dddf-4637-b485-7fa832467acc.jpg)
Inner surface:
![WhatsApp Image 2023-03-05 at 14 29 07](https://user-images.githubusercontent.com/1646078/222966713-21cce098-185f-42a9-bd5d-04909cba5f07.jpg)
Side surface:
![WhatsApp Image 2023-03-05 at 14 28 50](https://user-images.githubusercontent.com/1646078/222966719-a8d07a13-3d68-4aac-95c7-06d98104ad59.jpg)
Top surface:
![WhatsApp Image 2023-03-05 at 14 28 32](https://user-images.githubusercontent.com/1646078/222966721-f97efbfd-fcef-4823-a49b-2a1274a596a2.jpg)

**Reply by 1coderookie** [reply]
Bottom (and so the first layer) does look good in terms of z-offset. So if you don't like that texture, maybe try a glass bed instead as mentioned before. 
Inner side and top layer: at least in SuperSlicer you can choose the finish, dunno about Cura tho, maybe do a little research on that.
Side: looks like you need to do some more calibration as there are gaps and even a complete line missing. I'd bet on underextrusion, so do e-steps calibration, cleaning your hotend, check flowrate, ensure filament gets feeded w/o any interruptions due to a bad-moving spool (maybe print a ball bearing spool holder) etc. Many guides out there to find about this topic, but I personally would start with calibrating e-steps and cleaning the nozzle by making a cold pull at least.

**Reply by SteveBz** [reply]
Hi,

Yes, I'll think about the glass bed.

I can easily recalibrate the extruder.  Seems like the quickest fix.

I'll look into super slicer.

Thanks for all that.

Steve.

**Reply by 1coderookie** [reply]
There's also a set of calibration tools within SuperSlicer which you can/should run in the order it's mentioned there. Comes in handy.  Here's a quick overview of it: https://www.fabbaloo.com/news/hands-on-with-superslicers-calibration-set
Choose the "Arachne Version" of SuperSlicer btw and activate using Arachne in the settings, it really is an improvement.

**Reply by 1coderookie** [reply]
Btw: Do you have a link to the STL of that case?

**Reply by SteveBz** [reply]
Here:

https://www.thingiverse.com/thing:4994599

**Reply by 1coderookie** [reply]
Great, thanks!
Aahh ok, it's THAT one :) Didn't I link to that as well? If not then I forgot, at least I had it on my list, gonna check it, thx..
